### ISSUE TITLE
[Feature Request] Add Explicit to album folders

### ISSUE THREAD

**Issue by Gamer4Life2020** [DJDoubleD/QobuzDownloaderX-MOD#38]
# **🚀 Feature Request**

## **Is your feature request related to a problem? Please describe.**

*

---

## **Describe the solution you'd like**
Add Explicit to album folders like [E] ALBUM for sorting (OrpheusDL) or a way to only pick highest quality like streamrip

*

---

## **Describe alternatives you've considered**
I can use the others but I like the GUI and not CMD.

*

---

### **Additional context**

*

**Comment by orbittwz** [comment]
The suggestions in #36 should cover this as well, so this might be merged with it...
Thanks

**Comment by orbittwz** [comment]
yep same as #116 kinda... will think about how to implement this.
